source/node/introduction.rst

```diff
@@ -135,43 +134,50 @@ To use the perun-node, the following pre-requisites need to be met.
    (b) A local blockchain network started using geth node or
    (c) A connection to the ropsten testnet.
 
-Note: If using ropsten testnet, user should have keys corresponding to accounts
-funded on the testnet and needs to follow additional steps when generating
-configuration artifacts.
+.. note::
+   If using ropsten testnet, user should have keys corresponding to accounts
+   funded on the testnet and needs to follow additional steps when generating
+   configuration artifacts.
+
+.. note::
+ For the code blocks that appear in the following sections, if they are prefixed by
+
+ - **$**, execute them in terminal,
+ - **>**, execute them in perun-node cli.
```

Comment:
Please check the indentation of the note block
```suggestion
   For the code blocks that appear in the following sections, if they are prefixed by

   - **$**, execute them in terminal,
   - **>**, execute them in perun-node cli.
```

Reply:
I have now updated all indentations (for notes and code blocks) to consistently have 2 spaces.